app/routes/tasks.py

```diff
@@ -0,0 +1,94 @@
+from datetime import datetime
+from flask import Blueprint, jsonify, request
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from app.helpers import *
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+# POST /tasks Create a new task to the db and return the task body.
+@tasks_bp.route("", methods = ["POST"])
+def post_new_task():
+
+    request_body = request.get_json()
+
+    if "title" not in request_body or "description" not in request_body or "completed_at" not in request_body:
+        return {"details": "Invalid data"}, 400
```

Comment:
Good job enforcing the POST data requirements. We should be doing similar checks when PUTting a task as well. So we could also think about moving this check into another validation helper.

We could even think about adding a class method to `Task` to create a new `Task` using the dictionary from a request body

```python
    @classmethod
    def from_dict(values):
        # create a new task, and set the model values from the values passed in
        # be sure to validate that all required values are present, we could return `None` or raise an error if needed
        return new_task